test/column/colSpan.test.ts

```diff
@@ -0,0 +1,117 @@
+import { within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import type { Column } from '../../src';
+import { setup, getRows, getHeaderCells, getSelectedCell } from '../utils';
+
+describe('colSpan', () => {
+  function setupColSpanGrid() {
+    const columns: Column<Row>[] = [];
+    type Row = number;
+    const rows: readonly Row[] = [...Array(30).keys()];
+
+    for (let i = 0; i < 15; i++) {
+      const key = String(i);
+      columns.push({
+        key,
+        name: key,
+        frozen: i < 5,
+        colSpan(args) {
+          if (args.type === 'ROW') {
+            if (key === '2' && args.row === 2) return 3;
+            if (key === '4' && args.row === 4) return 6; // Will not work as colspan includes both frozen and regular columns
+            if (key === '0' && args.row === 5) return 5;
+            if (key === '6' && args.row < 8) return 2;
+          }
+          if (args.type === 'HEADER' && key === '8') {
+            return 3;
+          }
+          return undefined;
+        },
+        cellClass(row) {
+          if (
+            (key === '0' && row === 5)
+              || (key === '2' && row === 2)
+              || (key === '6' && row < 8)
+          ) {
+            return 'col-span';
+          }
+          return undefined;
+        }
+      });
+    }
+    setup({ columns, rows });
+  }
+
+  it('should merges cells', () => {
+    setupColSpanGrid();
+    // header
+    expect(getHeaderCells()).toHaveLength(13);
+
+    // rows
+    const gridRows = getRows();
+    const getCells = (rowIdx: number) => within(gridRows[rowIdx]).getAllByRole('gridcell');
```

Comment:
Should we add `getCellsAtRowIndex(rowIdx)` in the test utils?

Reply:
Added